Lib9c/Model/Item/Inventory.cs

```diff
@@ -275,6 +275,49 @@ public Item AddNonFungibleItem(ItemBase itemBase, ILock iLock = null)
             return nonFungibleItem;
         }
 
+        /// <summary>
+        /// Add tradable fungible item.
+        /// If there is same item which is not locked, add count to it.
+        /// The same item is item which has same item sheet id, fungible id, tradable id and
+        /// same required block index.
+        /// If there is no same item, add new item.
+        /// </summary>
+        /// <exception cref="ArgumentNullException" />
+        /// <exception cref="ArgumentOutOfRangeException" />
+        public Item AddTradableFungibleItem(
+            ITradableFungibleItem tradableFungibleItem,
+            int count)
+        {
+            if (tradableFungibleItem is null)
+            {
+                throw new ArgumentNullException(nameof(tradableFungibleItem));
+            }
+
+            Item item;
+            if (!TryGetTradableFungibleItems(
+                    tradableFungibleItem.FungibleId,
+                    tradableFungibleItem.RequiredBlockIndex,
+                    null,
+                    out var outItems))
+            {
+                item = new Item((ItemBase)tradableFungibleItem, count);
+                _items.Add(item);
+                return item;
+            }
+
+            item = outItems.First();
+            if (item.count > int.MaxValue - count)
+            {
+                throw new ArgumentOutOfRangeException(
+                    nameof(count),
+                    count,
+                    $"Aborted because {nameof(item.count)} is overflow");
+            }
+
+            item.count += count;
```

Comment:
I'm not sure we've validated `item.count` must be positive value before.

Reply:
As far as I understand, inventory items cannot have a count less than 1.🤔